wordpress-mobile/WordPress-iOS
# Table Views: use `insetGrouped` style

~While this PR is not big code-wise~ (can't say that anymore, oof), grab some coffee and sit tight because we have a few flows to double-check. :)

The main goal is to update **some** of the table views in the app to use the `insetGrouped`. This is a style added in iOS 13 with makes the app feel fresh and it also connects with the upcoming card designs we'll have for My Site Dashboard.

These are pure visual changes, there are no changes in functionality and/or structure.

| Before | After |
| ------ | ----- |
| ![IMG_7581BD87AE30-1](https://user-images.githubusercontent.com/7040243/151016254-dac772cb-f6a4-4b3f-9d32-3d8f8c32d515.jpeg) | ![Simulator Screen Shot - iPhone 13 - 2022-01-25 at 13 20 43](https://user-images.githubusercontent.com/7040243/151016334-98216a9f-d8d9-4333-a5cd-253a3f4ccc4c.png) |

## To test

The best way to test is to check these flows in iPad/iPhone in both light/dark mode. Check as many screens as possible, keep in mind though that not all of them were changed.

Some of them will keep the previous visual (like Activity Log) and some others couldn't be changed without causing errors (like tag picker). It's normal for many apps to use both styles, this is definitely not a problem. Even the "Phone" app use multiples.

### My Site

Check the new table view style and make sure that when scrolling the navbar changes its background when changing to the compact mode.

I also recommend 1) enabling Quick Start 2) enabling the redeem domain section (can be done on line 754 on `BlogDetailsViewController.m`) or using a site in which you have credits to redeem a domain.

#### My Site: Jetpack Settings

In a Jetpack Site (Atomic or Jetpack) go to Jetpack Settings. Check the table and also make sure to check deeper sections (like Allowlisted IP addresses and Manage Connection)

#### My Site: Sharing

Check the "Sharing" section and the deeper sections.

#### My Site: Site Settings

Go one by one checking this section and its deeper sections.

### Me

On "My Site" tap your avatar. Check this screen and all the sections inside.

## Regression Notes
1. Potential unintended areas of impact
There might be some other flows making use of those screens, displaying them using the `insetGrouped` style is not a problem.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual test.

3. What automated tests I added (or what prevented me from doing so)
Visual changes were tested manually.

PR submission checklist:
